ffi/Makefile.linux

```diff
@@ -2,7 +2,7 @@
 CXX ?= g++
 
 # -flto and --exclude-libs allow us to remove those parts of LLVM we don't use
-CXX_FLTO_FLAGS ?= -flto
+#CXX_FLTO_FLAGS ?= -flto
```

Comment:
Kindly ignore this. I was facing some difficulties building locally so had to turn LTO off until I sort that.

Reply:
```suggestion
CXX_FLTO_FLAGS ?= -flto
```